workspaces/ui-v2/src/optic-components/common/CommitMessageModal.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { ChangeEvent, FC, useState } from 'react';
```

Comment:
Pulled out from `/src/optic-components/diffs/render/AskForCommitMessage.tsx` and pulled out the generic UI part (since this is now used in two contexts)